presto-docs/README.md

```diff
@@ -0,0 +1,73 @@
+# presto-docs - Presto Documentation
+
+The presto-docs module contains the reference documentation for Presto.
+
+## Tools
+
+The engine used to create the documentation in HTML format is the Python-based
+[Sphinx](https://www.sphinx-doc.org). Installation instructions for various OS
+are [available on the Sphinx site](https://www.sphinx-doc.org/en/master/usage/installation.html).
+
+Documentation source files can be found in [Restructured
+Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
+`src/main/sphinx` and sub-folders.
+
+In addition, `make` is required
+
+## Build and Read
+
+With the tools installed and available on the PATH, you can build the docs
+easily with make:
+
+```bash
+cd presto/presto-docs
+make clean html
+```
+
+Alternatively you can keep using Apache Maven and Java like for the rest of the
+Presto build. You just need to have built the current version from the root.
+Subsequently you can build the site:
+
+```bash
+cd presto/presto-docs
+mvn clean install
+```
+
+It calls the make command and performs other checks, so you still need the tools
+installed.
+
+However you built the docs, the output HTML files can be found in the folder
+`presto-docs/target/html/`.
+
+You can open the file `presto-docs/target/html/index.html` in a web browser on
+macOS with
+
+```bash
+open presto-docs/target/html/index.html
+```
+
+Or you can directly call your browser of choice with the filename e.g on Ubuntu
+with Chromium:
+
+```bash
+chromium-browser presto-docs/target/html/index.html
```

Comment:
On Ubuntu you can use `xdg-open` to open in user's default browser

Reply:
added